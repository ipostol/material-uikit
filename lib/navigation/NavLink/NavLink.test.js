'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _NavLink = require('./NavLink');

var _NavLink2 = _interopRequireDefault(_NavLink);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('NavLink component', function () {

  it('should render correctly', function () {

    var props = {};

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_NavLink2.default, props));

    expect(wrapper).toMatchSnapshot();
  });

  it('should render correctly to props', function () {

    var props = {
      to: '/'
    };

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_NavLink2.default, props));

    expect(wrapper).toMatchSnapshot();
  });
});