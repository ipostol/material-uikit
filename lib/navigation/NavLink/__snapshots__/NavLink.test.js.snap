// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavLink component should render correctly 1`] = `
<Link
  className="general"
  onlyActiveOnIndex={false}
  style={Object {}}
>
  <ArrowLeft
    style={
      Object {
        "margin": "0px 16px",
      }
    }
  />
</Link>
`;

exports[`NavLink component should render correctly to props 1`] = `
<Link
  className="general bold"
  onlyActiveOnIndex={false}
  style={Object {}}
>
  <ArrowLeft
    style={
      Object {
        "margin": "0px 16px",
      }
    }
  />
</Link>
`;
